src/actions/archiveThought.ts

```diff
@@ -93,7 +93,8 @@ const archiveThought = (state: State, options: { path?: Path }): State => {
   /** Gets the previous sibling context in the context view. */
   const prevContext = () => {
     const thoughtsContextView = thoughtsEditingFromChain(state, path)
-    const contexts = showContexts ? getContextsSortedAndRanked(state, headValue(state, thoughtsContextView)) : []
+    const head = headValue(state, thoughtsContextView)
+    const contexts = showContexts && head ? getContextsSortedAndRanked(state, head) : []
```

Comment:
my thinking is that if head value is either `undefined` or `''`, we don't want to `getContextsSortedAndRanked` it, and just go with `[]`

Reply:
We should just check for `undefined`. Checking for empty string is outside the scope of this PR.

Reply:
ok, that totally makes sense, thanks
I've also changed all other places to check against `value !== undefined`, so we can allow "" thoughts